NAMESPACE

```diff
@@ -928,14 +932,12 @@ export(users_delete_me_favorites)
 export(users_delete_sessions)
 export(users_get)
 export(users_get_api_keys)
-export(users_get_me_themes)
 export(users_list)
 export(users_list_api_keys)
 export(users_list_me)
 export(users_list_me_activity)
 export(users_list_me_favorites)
 export(users_list_me_organization_admins)
-export(users_list_me_themes)
```

Comment:
Laura and I looked into the "themes" endpoints. It appears to be in the [civis python client](https://civis-python.readthedocs.io/en/stable/api_users_endpoint.html#civis.resources._resources.Users.list_me_themes) and the API docs on Platform, but did not see it when inspecting the public endpoints ([job](https://platform.civisanalytics.com/spa/#/scripts/custom/322660286)). They are also not listed in version 3.0.0 of the [civis-r docs](https://civisanalytics.github.io/civis-r/reference/index.html). Removing these seems alright given that they would be little used on the client, and the information is still available in `users_me`.